### DISCUSSION TITLE
Basic question about the example key layouts

### DISCUSSION THREAD

**Discussion by harryfrogfather** [meow-edit/meow#180] [Q&A] [answered]
I was checking the example layouts for inspiration on my own ( Colemak DH), and noticed that Qwerty Dvorak and Colemak have 
( number  . meow-digit-argument ) in the  **meow-leader-define-key** section , but Programmer Dvorak has these options in the **meow-normal-define-key** section . Is this an error or a different approach or my misunderstanding. 
Thanks for the great package.

**Comment by DogLooksGood** [comment, marked as answer]
Hi, it's intentionally designed like this.

Because to type a number in programmer dvorak, you have to hold shift.  Command `meow-expand-*`s are frequently used, so bind them to symbols(use without shift), then the number keys are free.

But the command layout is really up to you, configure it the way you like. You don't have to follow the examples.

**Reply by harryfrogfather** [reply]
LOLZ yes , obvious now you have pointed it out ( in my defense its been 15 years since I looked at programmer Dvorak).  I had set most of my config up, was just looking at the examples for the few keys I was uncertain about. Meow does seem to make my emacs faster than evil mode. Thanks for the quick reply.
